rust_snuba/src/processors/snapshots/rust_snuba__processors__tests__schemas@profiles-call-tree-0.snap

```diff
@@ -1,113 +1,115 @@
 ---
 source: src/processors/mod.rs
-description: "{\n  \"functions\": [\n    {\n      \"fingerprint\": 10027678537215769000,\n      \"function\": \"CursorWrapper._execute\",\n      \"package\": \"django.db.backends.utils\",\n      \"in_app\": false,\n      \"self_times_ns\": [\n        9968883, 9973116, 9973263, 19933212, 9968120, 9965574, 29905829,\n        129589001, 9967363, 19938577, 39881572, 11600464, 49840810, 19938549,\n        29913753, 9965667, 19945134, 10003759, 9966104, 9964477, 9969739,\n        20809886, 11830791\n      ]\n    },\n    {\n      \"fingerprint\": 888186900786406300,\n      \"function\": \"connect\",\n      \"package\": \"psycopg2\",\n      \"in_app\": false,\n      \"self_times_ns\": [19946707]\n    },\n    {\n      \"fingerprint\": 3734820929566501400,\n      \"function\": \"<module>\",\n      \"package\": \"__main__\",\n      \"in_app\": true,\n      \"self_times_ns\": [10289412]\n    }\n  ],\n  \"environment\": \"production\",\n  \"profile_id\": \"0f2d3a4efaa94234b0eb22b1ee7e69a5\",\n  \"platform\": \"python\",\n  \"project_id\": 1,\n  \"received\": 1687971966,\n  \"retention_days\": 90,\n  \"timestamp\": 1687971965,\n  \"transaction_name\": \"GET /api/1/events\",\n  \"transaction_op\": \"http.server\",\n  \"transaction_status\": \"ok\",\n  \"http_method\": \"GET\"\n}\n"
+description: "{\n  \"functions\": [\n    {\n      \"fingerprint\": 10027678537215769000,\n      \"function\": \"CursorWrapper._execute\",\n      \"package\": \"django.db.backends.utils\",\n      \"in_app\": false,\n      \"self_times_ns\": [\n        9968883, 9973116, 9973263, 19933212, 9968120, 9965574, 29905829,\n        129589001, 9967363, 19938577, 39881572, 11600464, 49840810, 19938549,\n        29913753, 9965667, 19945134, 10003759, 9966104, 9964477, 9969739,\n        20809886, 11830791\n      ]\n    },\n    {\n      \"fingerprint\": 888186900786406300,\n      \"function\": \"connect\",\n      \"package\": \"psycopg2\",\n      \"in_app\": false,\n      \"self_times_ns\": [19946707]\n    },\n    {\n      \"fingerprint\": 3734820929566501400,\n      \"function\": \"<module>\",\n      \"package\": \"__main__\",\n      \"in_app\": true,\n      \"self_times_ns\": [10289412]\n    }\n  ],\n  \"environment\": \"production\",\n  \"profile_id\": \"0f2d3a4efaa94234b0eb22b1ee7e69a5\",\n  \"platform\": \"python\",\n  \"project_id\": 1,\n  \"received\": 1687971966,\n  \"retention_days\": 90,\n  \"timestamp\": 1687971965,\n  \"transaction_name\": \"GET /api/1/events\"\n}\n"
 expression: snapshot_payload
 ---
 [
-  {
```

Comment:
that's not the right fix. those files must not change, and I think they do because you're passing a nested array to `InsertBatch` instead of a flat one